package.json

```diff
@@ -30,30 +20,33 @@
   },
   "scripts": {
     "start": "start-storybook -p 9001 -c storybook -s ./storybook/static",
-    "build": "lerna run --parallel build -- -- --env production && npm run ts:emit",
+    "build": "lerna run --parallel build -- --env production && yarn ts:emit",
     "build:storybook": "build-storybook -c storybook -o ./storybook/public -s ./storybook/static",
-    "format": "npm run prettier && npm run eslint:format",
-    "init": "npm install && lerna bootstrap && npm run build",
-    "link-all-packages": "lerna exec --ignore @leafygreen-ui/lib --ignore @leafygreen-ui/theme -- npm link ",
+    "format": "yarn prettier && yarn eslint:format",
+    "init": "yarn && yarn build",
+    "link-all-packages": "lerna exec --ignore @leafygreen-ui/lib --ignore @leafygreen-ui/theme -- yarn link",
     "link": "./scripts/link.sh",
-    "lint": "npm run prettier:check && npm run eslint && npm run ts:check",
+    "lint": "yarn prettier:check && yarn eslint && yarn ts:check",
     "eslint": "eslint \"**/*.{js,ts,tsx}\"",
     "eslint:format": "eslint \"**/*.{js,ts,tsx}\" --fix",
     "prettier": "prettier --write \"**/*.{js,ts,tsx,json,md}\"",
     "prettier:check": "prettier --check \"**/*.{js,ts,tsx,json,md}\"",
     "test": "jest --env jsdom",
     "ts:check": "tsc --project tsconfig.json --noEmit",
     "ts:emit": "lerna run ts:emit",
-    "prepublishOnly": "npm run lint && npm run test && npm run build",
+    "prepublishOnly": "yarn lint && yarn test && yarn build",
     "release": "lerna publish --no-commit-hooks",
     "release:site": "gh-pages --dist storybook/public"
   },
   "husky": {
     "hooks": {
       "commit-msg": "commitlint -E HUSKY_GIT_PARAMS",
-      "pre-push": "npm run lint"
+      "pre-push": "yarn lint"
     }
   },
+  "peerDependencies": {
+    "@emotion/core": "^10.0.10"
```

Comment:
The warning was saying that this IS a peerDependency. i.e. we need to install it ourselves in devDependencies.

Reply:
👍